gvm/connections.py

```diff
@@ -305,6 +305,31 @@ def connect(self):
 
 
 class DebugConnection:
+    """Wrapper around a connection for debugging purposes
+
+    Allows to debug the connection flow including send and read data. Internally
+    it uses the python `logging`_ framework to create debug messages. Please
+    take a look at `the logging tutorial <https://docs.python.org/3.5/howto/logging.html#logging-basic-tutorial>`_
```

Comment:
If this is not intended to be specific to Python 3.5, the following URL would be more appropriate:

https://docs.python.org/3/howto/logging.html#logging-basic-tutorial